docs/_fr/user/guides/canaux.md

```diff
@@ -275,6 +278,32 @@ style="width: 100px;">
 Pour en savoir plus sur ce connecteur, voir ses sources et son _README_ dans le dossier 
 [connector-google-chat](https://github.com/theopenconversationkit/tock/tree/master/bot/connector-google-chat) sur GitHub.
 
+### Google Chat
+
+<img alt="Logo Google Chat" title="Google Chat"
+src="https://cdn.zapier.com/storage/photos/bfbce5bee25b1b50d8a910c30588c61e.png"
+style="width: 100px;">
+
+* **Canal** : [Google Chat](https://chat.google.com) (anciennement Google Hangouts)
+* **Type** : texte
+* **Status** : connecteur Tock utilisé hors production
+
+Pour en savoir plus sur ce connecteur, voir ses sources et son _README_ dans le dossier
+[connector-google-chat](https://github.com/theopenconversationkit/tock/tree/master/bot/connector-google-chat) sur GitHub.
+
+### Mattermost
+
+<img alt="Logo Mattermost" title="Mattermost"
+src="../../img/mattermost.svg"
```

Comment:
In the new documentation version paths to static ressources are fixed now :)  I think this path will change with the new file location : 
```suggestion
src="../img/mattermost.svg"
```